src/formSchema/index.ts

```diff
@@ -6,15 +6,16 @@ export const tagOptionSchema = z.object({
 })
 
 export const assetFormSchema = z.object({
-  altText: z.string().trim().optional(),
+  alt: z.string().trim().optional(),
```

Comment:
This should improve. don't use the library  name as `z`
Also, this should be wrap with util